runtime/near-vm-logic/src/mocks/mock_external.rs

```diff
@@ -209,6 +248,46 @@ impl External for MockedExternal {
     fn validator_total_stake(&self) -> Result<Balance> {
         Ok(self.validators.values().sum())
     }
+
+    #[cfg(feature = "protocol_feature_function_call_weight")]
+    fn distribute_unused_gas(&mut self, gas: Gas) -> GasDistribution {
```

Comment:
comment (especially the part that you're actually modifying the assigned_gas for the receipts

Reply:
This is documented on the trait. Are you suggesting all implementations to give details about how specifically it distributes gas?

Reply:
yes, if possible. It helps a lot during code review and later code reading.